cidc_api/utils/migrations.py

```diff
@@ -47,10 +48,28 @@ def run_all(self):
                 raise
 
 
+@contextmanager
+def migration_session():
+    session = Session(bind=op.get_bind())
+
+    try:
+        yield session
+    except:
+        session.rollback()
+        raise
+    finally:
```

Comment:
> The finally clause runs whether or not the try statement produces an exception.

I think an `else` should be used here.

Reply:
Actually, according to the SQLAlchemy docs, we want:
```python
    try:
        yield session
        session.commit()
    except:
        session.rollback()
        raise
    finally:
        session.close()
```
Whoops 😅